content/guides/agent_checks.md

```diff
@@ -476,26 +476,34 @@ Troubleshooting
 ======================================================
 -->
 
-<h3 id="troubleshooting">Troubleshooting</h3>
+### Troubleshooting
 
 Custom Agent checks can't be directly called from python and instead
  need to be called by the agent. To test this, run:
 
-    sudo -u dd-agent dd-agent check my_check
+    sudo -u dd-agent dd-agent check <CHECK_NAME>
 
 If your issue continues, please reach out to Support with the help page that
  lists the paths it installs.
 
-<h4>Testing Custom Checks on Windows</h4>
+#### Testing Custom Checks on Windows
 
+* **For agent version < 5.12**:<br>
 Testing custom checks on Windows is easy. The Agent install includes a file called shell.exe
 in your Program Files directory for the Datadog Agent which you can use to run python within the Agent environment.
-
-Once your check (called "my_check") is written and you have the .py and .yaml files
+Once your check (called "`<CHECK_NAME>`") is written and you have the .py and .yaml files
```

Comment:
(called `<CHECK_NAME>`)

(no double quotes, as <CHECK_NAME> is already in code ticks.